src/content/api/media-uploads.mdoc

```diff
@@ -44,3 +44,84 @@ When the Media Upload was updated.
 The User or API Key that updated the Media Upload.
 {% /property %}
 {% /properties %}
+
+{% endpoint method="POST" path="/api/media-uploads" %}
+## Create a presigned URL for Media Upload EXPERIMENTAL
+{% row %}
+{% column %}
+### Required Fields
+{% properties %}
+{% property name="accountId" type="string" %}
+The Media Upload's owning Centrapay Account id.
```

Comment:
Comment: I wonder if we could define the fields, types and descriptions in one place so that we don't need so much repetition. We would have to flag which routes they are used in though 🤔